k8s/ietfweb/wagtail.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: wagtail
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: wagtail
+  template:
+    metadata:
+      labels:
+        app: wagtail
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+        # -----------------------------------------------------
+        # wagtail Container
+        # -----------------------------------------------------
+        - name: wagtail
+          image: "ghcr.io/ietf-tools/www:$APP_IMAGE_TAG"
+          imagePullPolicy: Always
```

Comment:
`IfNotPresent`?

Reply:
This is consistent with https://github.com/ietf-tools/datatracker/blob/main/k8s/datatracker.yaml#L42
May be it's better to have it pulled always since we control the image generation and we know things can go wrong with releases?

Reply:
Sure, that's fine by me.